Dockerfile

```diff
@@ -24,7 +22,7 @@ RUN go mod vendor
 RUN make build
 
 # Use OpenShift base image
-FROM ${TARGET_IMAGE}
+FROM registry.access.redhat.com/ubi9/ubi-minimal:latest
```

Comment:
FWIW we're currently discussing to switch to `9.5` instead of `latest`.

Reply:
I can change it when we settle on this - I had the feeling the decision was not over :-)